app/models/account.rb

```diff
@@ -63,6 +63,14 @@ def self.single_tenant_default
     end
   end
 
+  # @return [Boolean] whether this Account is the global tenant in a multitenant environment
+  def self.global_tenant?
+    # Global tenant only exists when multitenancy is enabled and NOT in test environment
+    # (Test environment always acts like a single tenant, i.e. tenant switching not possible)
```

Comment:
Thanks @atz. Aha, yes, I didn't mean to state/imply `:singletenant` is the default (as I realize it isn't). The important point in the comment is the part at the end: `tenant switching is currently not possible`, and that is the reason why a global_tenant doesn't make any sense in the test environment (i.e. `global_tenant?==false`). 

Would it be reasonable to replace this last line with simply saying: 
`(In test environment tenant switching is currently not possible)`